fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -2724,6 +2732,73 @@ public static short calcShortKeyColumnCount(List<Column> columns, Map<String, St
         return shortKeyColumnCount;
     }
 
+    public static short calcShortKeyColumnCount(List<Column> indexColumns, Map<String, String> properties,
```

Comment:
Can we use calcShortKeyColumnCount(x, y) to implement this function，avoid duplicated code.

Reply:
yes, changed.